example_school_library_decorator/person.rb

```diff
@@ -4,12 +4,12 @@ class Person
   attr_accessor :name, :age
   attr_reader :id
 
-  def initialize(age, name = 'Unknown', is_parent_permitted: true)
+  def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @corrector = Corrector.new
     @age = age
     @name = name
-    @parent_permission = is_parent_permitted
+    @parent_permission = parent_permission
   end
```

Comment:
- [ ] Let's refactor the initializer here so that we may be able to store rentals
```suggestion
def initialize(age, name = 'Unknown', parent_permission: true)
    @id = Random.rand(1..1000)
    @corrector = Corrector.new
    @age = age
    @name = name
    @parent_permission = parent_permission
    @rentals = []
  end
```

- [ ] We will also need to add a method for adding rentals inside this Person class
```ruby
def add_rental(book, date)
  Rental.new(date, book, self)
end
```